pages/examples/withplatform/geoProcessGraphic.html

```diff
@@ -0,0 +1,231 @@
+                        "boundary": {
+                          "span": "P30D"
+                        }
+                    }
+                },
+                AirTempValueBoat2: {
+                  kind: 'numeric',
+                  value: { tsx: `$event['amb_air_temp'].Double${- 5}` },
+                  aggregation: {
+                    "tsx": "right($value)"
+                  },
+                  interpolation: {
+                      "kind": "linear",
+                      "boundary": {
+                        "span": "P30D"
+                      }
+                  }
+                }
+              },
+              { },
+              {
+                positionXVariableName : 'LatitudeValueBoat2',
+                positionYVariableName: 'LongitudeValueBoat2',
+                alias: 'Titanic2', 
+                image: '/images/boat.png'
+              }
+            );
+
+            let tsqExpression3 = new tsiClient.ux.TsqExpression(
+              {timeSeriesId: ['9461776'] },
+              {
+                LatitudeValueBoat3: {
+                    kind: 'numeric',
+                    value: { tsx: `$event.lat.Double${- 10}`},
+                    aggregation: {
+                      "tsx": "right($value)"
+                    },
+                    interpolation: {
+                        "kind": "linear",
+                        "boundary": {
+                          "span": "P30D"
+                        }
+                    }
+                },
+                LongitudeValueBoat3: {
+                    kind: 'numeric',
+                    value: { tsx: `$event.lon.Double${- 10}`},
+                    aggregation: {
+                      "tsx": "right($value)"
+                    },
+                    interpolation: {
+                        "kind": "linear",
+                        "boundary": {
+                          "span": "P30D"
+                        }
+                    }
+                },
+                AirTempValueBoat3: {
+                  kind: 'numeric',
+                  value: { tsx: `$event['amb_air_temp'].Double${- 10 }` },
+                  aggregation: {
+                    "tsx": "right($value)"
+                  },
+                  interpolation: {
+                      "kind": "linear",
+                      "boundary": {
+                        "span": "P30D"
+                      }
+                  }
+                }
+              },
+              { },
+              {
+                positionXVariableName : 'LatitudeValueBoat3',
+                positionYVariableName: 'LongitudeValueBoat3',
+                alias: 'Titanic3', 
+                image: '/images/boat.png'
+              }
+            );
+
+
+            let image =  '';
+            geoProcessGraphic.render(environmentFqdn, authContext.getTsiToken, image, [tsqExpression1,tsqExpression2,tsqExpression3], geoProcessGraphicOptions);
```

Comment:
oof yeah i see why we need this because it implements history playback, really wondering if we can avoid that but im not sure that multiple function signatures for the same name are possible

Reply:
made the changes so it's no longer needed